### PR TITLE
Add IoT Hub - Azure Digital Twins - Time Series Insights sample

### PR DESCRIPTION
**Pull request by algattik**
Implement a stream processing architecture using:

- IoT Hub (Ingest)
- Azure Digital Twins (Model Management / Stream Process / Routing)
- Time Series Insights (Serve / Store to Parquet)
- Azure Storage (Serve for data analytics)

### CODE REVIEW

File: components/terraform/iothub/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+output "listen_event_hubs_primary_connection_string" {
+  value       = "Endpoint=${azurerm_iothub.main.event_hub_events_endpoint};SharedAccessKeyName=${local.listenPolicy};SharedAccessKey=${azurerm_iothub.main.shared_access_policy[index(azurerm_iothub.main.shared_access_policy.*.key_name, local.listenPolicy)].primary_key};EntityPath=${azurerm_iothub.main.event_hub_events_path}"
+  description = "The primary connection string to receive events."
+  sensitive   = true
+}
+
+output "registry_write_primary_connection_string" {
+  value       = "HostName=${azurerm_iothub.main.hostname};SharedAccessKeyName=${local.registryWritePolicy};SharedAccessKey=${azurerm_iothub.main.shared_access_policy[index(azurerm_iothub.main.shared_access_policy.*.key_name, local.registryWritePolicy)].primary_key}"
+  description = "The primary connection string to send events."
```
Comment (marcgs) [inline components/terraform/iothub/outputs.tf:28]: nit: The primary connection string to register devices

**Comment by marcgs** [comment]
Works like a charm when calling `create-solution.sh` on my Mac 👍

**Review by marcgs — APPROVED** [review]

File: iothub-digitaltwins-timeseriesinsights/README.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+and Azure Time Series Insights as database to store JSON data.
+Time Series Insights ingests event data from Event Hubs and makes it available
+for online querying and exploration. In addition, it stores the data in Parquet
+format on Azure Storage, where it is available for analytics.
+
+The provided scripts will create an end-to-end solution complete with load test client.
+
+## Running the Scripts
+
+Please note that the scripts have been tested on [Ubuntu 18 LTS](http://releases.ubuntu.com/18.04/), so make sure to use that environment to run the scripts. You can run it using Docker, WSL or a VM:
```
Comment (marcgs) [inline iothub-digitaltwins-timeseriesinsights/README.md:31]: nit: the scripts run natively on a Mac as well

File: iothub-digitaltwins-timeseriesinsights/create-solution.sh [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+    RUN=`echo $STEPS | grep M -o || true`
+    if [ ! -z "$RUN" ]; then
+        export IOTHUB_RESOURCES=$(terraform output -raw iothub_resource_id)
+        export EVENTHUB_NAMESPACES=$(terraform output -raw eventhub_namespace_names)
+        source ../components/azure-event-hubs/report-throughput.sh
+    fi
+echo
+
+echo "***** [V] Starting deployment VERIFICATION"
```
Comment (marcgs) [inline iothub-digitaltwins-timeseriesinsights/create-solution.sh:159]: What is this step doing exactly? I could not find any information on this step in the README.

File: components/terraform/iothub/outputs.tf (hunk 1 above)
Comment (algattik) [reply components/terraform/iothub/outputs.tf]: Removed this variable altogether

File: iothub-digitaltwins-timeseriesinsights/create-solution.sh (hunk 3 above)
Comment (algattik) [reply iothub-digitaltwins-timeseriesinsights/create-solution.sh:182]: Added comment in README

File: iothub-digitaltwins-timeseriesinsights/README.md (hunk 2 above)
Comment (algattik) [reply iothub-digitaltwins-timeseriesinsights/README.md:31]: Added

**Pull request merged**